sopel/web.py

```diff
@@ -86,7 +86,7 @@ def get(uri, timeout=20, headers=None, return_headers=False,
         headers = default_headers
     else:
         tmp = default_headers.copy()
-        headers = tmp.update(headers)
+        tmp.update(headers)
```

Comment:
Just saw the reason why: #1091 .  Just set the same functionality for all instances of tmp.update(headers)